Missing include leads to compilation error

Type `field_element` is used [here](https://github.com/NilFoundation/crypto3-zk-marshalling/blob/f22ae64e0e60e17d9f731df51b2da641a896590f/include/nil/crypto3/marshalling/zk/types/plonk/assignment_table.hpp#L58), but not included.

As a result, if you include this file:

```cpp
#include "nil/crypto3/marshalling/zk/types/plonk/assignment_table.hpp"
```

you will get compilation error:

```
.../include/nil/crypto3/marshalling/zk/types/plonk/assignment_table.hpp:58:29: error: 'field_element' was not declared in this scope
   58 |                             field_element<TTypeBase, typename PlonkTable::field_type::value_type>,
      |                             ^~~~~~~~~~~~~
```

Probably this include needs to be added to this file:

```cpp
#include <nil/crypto3/marshalling/algebra/types/field_element.hpp>
```

--- a/include/nil/crypto3/marshalling/zk/types/plonk/assignment_table.hpp
+++ b/include/nil/crypto3/marshalling/zk/types/plonk/assignment_table.hpp
@@ -36,6 +36,7 @@
 #include <nil/marshalling/types/integral.hpp>
 #include <nil/marshalling/status_type.hpp>
 #include <nil/marshalling/options.hpp>
+#include <nil/crypto3/marshalling/algebra/types/field_element.hpp>
 
 namespace nil {
     namespace crypto3 {

